libs/expo/betterangels/src/lib/ui-components/ClientCard.tsx

```diff
@@ -1,24 +1,33 @@
 import {
+  IdCardOutlineIcon,
   LocationDotIcon,
   TentIcon,
   ThreeDotIcon,
+  UserOutlineIcon,
 } from '@monorepo/expo/shared/icons';
 import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
+import {
+  Avatar,
+  IconButton,
+  TextBold,
+  TextButton,
+  TextRegular,
+} from '@monorepo/expo/shared/ui-components';
+import { format } from 'date-fns';
 import { useRouter } from 'expo-router';
 import { DimensionValue, Pressable, StyleSheet, View } from 'react-native';
-import Avatar from '../Avatar';
-import IconButton from '../IconButton';
-import TextBold from '../TextBold';
-import TextButton from '../TextButton';
-import TextRegular from '../TextRegular';
-
+import { HmisProfileType } from '../apollo';
 type TSpacing = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
-
 interface IClientCardProps {
   imageUrl?: string;
```

Comment:
as we moved under betterangels path can we type from generated types? 


besides progress and arrivedFrom everything can be ClientProfile I guess

Reply:
good call. updated